enf/dns_zone_test.go

```diff
@@ -0,0 +1,205 @@
+package enf
+
+import (
+	"context"
+	"net/http"
+	"testing"
+	"time"
+)
+
+func TestDNSService_ListDNSZones(t *testing.T) {
```

Comment:
`TestDNSService_ListDNSZones` -> `TestDNSService_ListZones`.  Ditto for the other test methods.

Reply:
My bad for missing these. They are fixed in the latest commit.